img_publisher:
  ros__parameters:
    video_file_path: "/data/YARGI.mp4" # Path to the video file relative to the package share directory
    publishing_frequency: 31 # Hz
    publishing_topic: "/filtered_images"

img_filterer:
  ros__parameters:
    subscribing_topic: "/filtered_images"
    gaussian_blur:
      enable: true
      publish: true
      publish_topic: "/filtered_images/gaussian_blur"
      show_image: true
      kernel_size: 9
    grayscale_conversion:
      enable: true
      publish: true
      publish_topic: "/filtered_images/grayscale"
      show_image: true
    canny_edge_detection:
      enable: true
      publish: true
      publish_topic: "/filtered_images/canny_edge"
      show_image: true
      low_threshold: 50
      high_threshold: 150